Makefile

```diff
@@ -24,6 +24,9 @@ dependencies:
 	# We use 'go get' here because 'go install' will not build out of our vendored mock repo.
 	go get github.com/golang/mock/mockgen
 
+	# Necessary to generate thrift files, in addition to manually installing thrift via homebrew
+	go get github.com/apache/thrift/lib/go/thrift
```

Comment:
Hrm, will this cause a problem because it's different than the vendored version?